app/helpers/handlers.js

```diff
@@ -50,24 +50,6 @@ const getPage = async (question, request, h) => {
   }
 
   switch (url) {
-  // reset environmentalImpact yar key if switching between pages
-    case 'roof-solar-PV':
```

Comment:
why was this removed?

Reply:
why it was there in the first place handled in payload depending on user response

Reply:
we added it as a user could potentially go through upgrading-existing-building etc, then go back to roof-solar-pv and go directly to solar-pv-system, so it was a way to reset all yarkeys on the branching paths